guides/deploy/README.md

```diff
@@ -218,51 +224,413 @@ You should see a response similar to:
+state: open
+```
+
+## View your Bids
+
+After a short time, you should see bids from providers for this deployment with the following command:
+
+```sh
+akash query market bid list --owner=$ACCOUNT_ADDRESS --node $AKASH_NODE --dseq $DSEQ
+```
+
+You should see a response similar to:
+
+```
+bids:
+- bid:
+    bid_id:
+      dseq: "140324"
+      gseq: 1
+      oseq: 1
+      owner: akash1vn06ycjjnvsvl639fet9lajjctuturrtx7fvuj
+      provider: akash10cl5rm0cqnpj45knzakpa4cnvn5amzwp4lhcal
+    created_at: "140326"
+    price:
+      amount: "1"
+      denom: uakt
+    state: open
+  escrow_account:
+    balance:
+      amount: "50000000"
+      denom: uakt
+    id:
+      scope: bid
+      xid: akash1vn06ycjjnvsvl639fet9lajjctuturrtx7fvuj/140324/1/1/akash10cl5rm0cqnpj45knzakpa4cnvn5amzwp4lhcal
+    owner: akash10cl5rm0cqnpj45knzakpa4cnvn5amzwp4lhcal
+    settled_at: "140326"
+    state: open
+    transferred:
+      amount: "0"
+      denom: uakt
+- bid:
+    bid_id:
+      dseq: "140324"
+      gseq: 1
+      oseq: 1
+      owner: akash1vn06ycjjnvsvl639fet9lajjctuturrtx7fvuj
+      provider: akash1f6gmtjpx4r8qda9nxjwq26fp5mcjyqmaq5m6j7
+    created_at: "140326"
+    price:
+      amount: "1"
+      denom: uakt
+    state: open
+  escrow_account:
+    balance:
+      amount: "50000000"
+      denom: uakt
+    id:
+      scope: bid
+      xid: akash1vn06ycjjnvsvl639fet9lajjctuturrtx7fvuj/140324/1/1/akash1f6gmtjpx4r8qda9nxjwq26fp5mcjyqmaq5m6j7
+    owner: akash1f6gmtjpx4r8qda9nxjwq26fp5mcjyqmaq5m6j7
+    settled_at: "140326"
+    state: open
+    transferred:
+      amount: "0"
+      denom: uakt
+
+```
+
+Note that there are bids from multiple different providers.  In this case, both providers happen to be willing to accept a price of *1 uAKT*. This means that the lease can be created using *1 uAKT* or *0.000001 AKT* per block to execute the container.
 
+For this example, we will choose `akash1f6gmtjpx4r8qda9nxjwq26fp5mcjyqmaq5m6j7`.
+
+For convenience and clarity for future referencing, we can extract the below value to a shell variable that we will be using to reference the deployment:
+
+| Attribute | Value |
+| --- | --- |
+| `PROVIDER` | `akash1f6gmtjpx4r8qda9nxjwq26fp5mcjyqmaq5m6j7` |
+
+Verify we have the right value populated by running:
+
+```sh
+echo $PROVIDER
+
+akash1f6gmtjpx4r8qda9nxjwq26fp5mcjyqmaq5m6j7 
 ```
 
-In this step, you post your deployment, and the Akash marketplace matches you with a provider via auction - the URI in the output is randomly generated. To create a deployment use akash deployment. The syntax for the deployment is `akash deploy create <config-path> --from <key-name>`. 
+## Create your Lease
+
+Create a lease for the bid from the chosen provider above by running:
+
+```sh
+akash tx market lease create --chain-id $AKASH_CHAIN_ID --node $AKASH_NODE --owner $ACCOUNT_ADDRESS --dseq $DSEQ --gseq $GSEQ --oseq $OSEQ --provider akash1f6gmtjpx4r8qda9nxjwq26fp5mcjyqmaq5m6j7 --from $KEY_NAME –fees 5000uakt
```

Comment:
This has that weird typo where `--fees` has that unicode character instead of two dashes